lib/instrumentation/undici.js

```diff
@@ -66,18 +66,21 @@ module.exports.unsubscribe = function unsubscribe() {
  * @param {Shim} shim instance of shim
  * @returns {TraceSegment} parent segment
  */
-function getParentSegment(shim) {
+function getParentContext(shim) {
```

Comment:
I think the doc block could use some language updating. It's referring to the segment and context.